static/css/zamboni/reviewers.less

```diff
@@ -261,10 +267,8 @@ ul.tabnav li a {
     padding: 5px 10px;
     border: 1px solid transparent;
     border-bottom: none;
-    -moz-border-radius-topleft:5px;
```

Comment:
the following changes are just fixes for obsolete properties I had warnings about